swap/tests/harness/mod.rs

```diff
@@ -234,6 +234,7 @@ fn start_alice(
         db,
         FixedRate::default(),
         bitcoin::Amount::ONE_BTC,
+        false,
```

Comment:
Without IDE type hints, it is practically impossible to know what this boolean does. Can we please model this in a clearer way?